### ISSUE TITLE
No default-plan

### ISSUE THREAD

**Issue by ArtrenH** [vplan-fr/VPlan_FR#47]
It would be nice to have a prediction for non-available dates based on past plans with no changes.

Also, if we want to create more advanced features in the future, this could be useful for a kind of digital calendar with user-based (shareable) custom events and notes for lessons.

**Comment by Belissimo-T** [comment]
@ArtrenH I think that these ideas have massive potential

**Comment by ArtrenH** [comment]
@Belissimo-T Are you planning to implement such a default plan compilation?

**Comment by Belissimo-T** [comment]
@ArtrenH Yes

**Comment by Belissimo-T** [comment]
depends on #169

**Comment by Belissimo-T** [comment]
fixed with c70f2c9
